src/server/sendMail.ts

```diff
@@ -5,24 +5,21 @@ import { CustomerStatusComponent } from '../app/customer-status/customer-status.
 import { callRivhit } from '../app/create-invoice/invoice';
 
 export async function sendMail(documentNumber: number, documentType: number) {
-  const { EMAIL_ADDRESS, EMAIL_PASSWORD } = process.env;
+  const { STORAGE_ROOM_EMAIL_ADDRESS, EMAIL_ADDRESS, EMAIL_PASSWORD } = process.env;
   const link = await CustomerStatusComponent.openDocument(documentType, documentNumber);
   const details = await getDocumentDetailsFromRivhit(documentType, documentNumber);
   const customer = await callRivhit("Customer.Get", { customer_id: details.customer_id });
-
+  
   if (!customer?.email) {
```

Comment:
This condition is handle the error in case there is no email. @noam-honig

Reply:
Hi Yosef, in this case, also an email will not be sent to the storage room - and I think that an email should be sent to the storage room in any case - no?

Reply:
You're right, I'll finish it and re-upload a pull request.